micro_epsilon_scancontrol_driver/include/micro_epsilon_scancontrol_driver/driver.h

```diff
@@ -47,6 +49,9 @@ namespace scancontrol_driver
             // Device setting functions
             int GetFeature(unsigned int setting_id, unsigned int *value);
             int SetFeature(unsigned int setting_id, unsigned int value);
+            // specialised features specific to encoding/decoding pattern
+            int SetTime(unsigned int setting_id, unsigned int value);
+            int GetTime(unsigned int setting_id, unsigned int* value);
```

Comment:
Both idle time and exposure time use common messages (because they have similar encoding/decoding structure). That's why the general named function.